pdns/syncres.hh

```diff
@@ -610,6 +610,13 @@ public:
     return old;
   }
 
+  uint32_t setRefreshAlmostExpired(bool doit)
```

Comment:
why is the return type `uint32_t` and not `bool`? (besides, the one caller in taskqueue.cc does not use the return value)

Reply:
fixed